doc/design/proposals/telemetry.md

```diff
@@ -0,0 +1,20 @@
+# Telemetry Support
+
+This is the design proposal for vSphere Integrated Containers Engine's initial integration with Vmware Analytics Cloud.  This data provides Vmware with simple usage metrics.  Currently, vSphere Integrated Containers Engine will piggyback on one of the data field that vSphere 6 and 6.5 are already collecting, specifically the OS full name used in the VM.  By providing a custom OS name for both the VCH and container VM, Vmware can determine the install base of VIC, the distribution of version installed, and the number of container VM used in a VCH.
+
+In future updates, VIC will be updated to collect data on which features are most commonly used to guide Vmware development plans.
+
+## Design Proposal
+
+vSphere collects a VM's OS full name by querying this data from the guest tools running inside the VM.  VIC installs a minimal toolbox inside the VCH's appliance VM and all container VMs created by VIC.  The toolbox will be updated to provide the following OS name for both VMs:
+
+| VM Type | OS Full Name |
+| --- | --- |
+| VCH | Photon OS - VCH v.v.v |
```

Comment:
Don't think there's any need to have PhotonOS in here given we are deploying an appliance. We could/?should? name it inline with the docs:
`VIC endpoint v.v.v`
Same for the containerVM OS:
`VIC containerVM v.v.v`